pkgs/development/python-modules/uptime-kuma-api/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, python-socketio
+, websocket-client
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "uptime-kuma-api";
+  version = "0.9.0";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    pname = "uptime_kuma_api";
+    inherit version;
+    hash = "sha256-nbRBsG4Mietd6kcrvpbTbWUP0SMRXbW4mWme1G5n3ks=";
+  };
+
+  propagatedBuildInputs = [
+    python-socketio
+    websocket-client
```

Comment:
why is this needed?
maybe you want `python-socketio.optional.client` or something like this?

Reply:
indeed, your suggestion is working